contrib/builder/rpm/armhf/generate.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+# vim: set ts=4 sw=4 noet :
+
+set -e
+
+# usage: ./generate.sh [versions]
+#    ie: ./generate.sh
+#        to update all Dockerfiles in this directory
+#    or: ./generate.sh centos-7
+#        to only update centos-7/Dockerfile
+#    or: ./generate.sh fedora-newversion
+#        to create a new folder and a Dockerfile within it
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+versions=( "$@" )
+if [ ${#versions[@]} -eq 0 ]; then
+	versions=( */ )
+fi
+versions=( "${versions[@]%/}" )
+
+for version in "${versions[@]}"; do
+	distro="${version%-*}"
+	suite="${version##*-}"
+	from="${distro}:${suite}"
+	installer=yum
+
+	if [[ "$distro" == "fedora" ]]; then
+		installer=dnf
+	fi
+	if [[ "$distro" == "photon" ]]; then
+		installer=tdnf
+	fi
+
+	mkdir -p "$version"
+
+	case "$from" in
+		centos:*)
+			# get "Development Tools" packages dependencies
+			image="multiarch/centos:7.2.1511-armhfp-clean"
+			;;
+		*)
+			image="${from}"
+			;;
+	esac
+
+	echo "$version -> FROM $image"
+	cat > "$version/Dockerfile" <<-EOF
+		#
+		# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/armhf/generate.sh"!
+		#
+
+		FROM $image
+	EOF
+
+	echo >> "$version/Dockerfile"
+
+	extraBuildTags='pkcs11'
+	runcBuildTags=
+
+	case "$from" in
+		oraclelinux:6)
```

Comment:
Have you tested the non-centos cases on arm? I'm thinking it might make sense to remove all of the other ones, (just have a centos and a default * case)  just so it's obvious what's supported and has been tested and what hasn't.